### ISSUE TITLE
guwen-punc模型中在transformers怎么使用？

### ISSUE THREAD

**Issue by jiangliqin** [Ethan-yt/guwen-models#1]
加载tokenizer 和 model后怎么使用？
    from transformers import AutoTokenizer, AutoModelForTokenClassification

    tokens = tokenizer(text, return_tensors='pt')
    logits = model(**tokens).logits
    tags = logits.argmax(axis=2).tolist()[0]
    得到的tag:0 1 2代表什么意思？0是原始token,1是逗号，2是句号吗？

可是huggingface显示得到这种格式：
[{'entity_group': ',', 'score': 0.9501149654388428, 'word': '赞', 'start': 5, 'end': 6}, {'entity_group': '.', 'score': 0.7064194679260254, 'word': '少', 'start': 12, 'end': 13}, {'entity_group': '.', 'score': 0.6454735398292542, 'word': '。', 'start': 13, 'end': 14}]
